src/javascript/app/base/header.js

```diff
@@ -649,6 +656,10 @@ const Header = (() => {
     };
 
     const logoutOnClick = async () => {
+        window.fcWidget?.user.clear().then(
+            () => window.fcWidget.destroy(),
```

Comment:
freshchat doesnt work when you are logged out? Or just clearing previous user data?